lib/mpl_toolkits/mplot3d/proj3d.py

```diff
@@ -124,6 +124,15 @@ def persp_transformation(zfront, zback):
                      [0,0,-1,0]
                      ])
 
+def ortho_transformation(zfront, zback):
+    a = -(zfront + zback)
+    b = -(zfront - zback)
+    return np.array([[2,0,0,0],
+                     [0,2,0,0],
+                     [0,0,-2,0],
+                     [0,0,a,b]
+                     ])
```

Comment:
Sorry, I should have justified it.
This matrix is almost equivalent to the one in the Wikipedia page. Based on `persp_transformation`, I made these assumptions:
- `zfront` is the value for `near`, `left` and `bottom`
- `zback` for `far`, `right`, `top`  
- the X and Y translations are omitted (I guess so the image will always be centered at the origin)

It's a simple derivation from the one from Wikipedia (v1), but since v2 and v1 are equivalent in homogeneous coordinates, I thought the second will look nicer.
<a href="https://www.codecogs.com/eqnedit.php?latex=$$&space;\vec{v_1}&space;=&space;\begin{bmatrix}&space;\frac{2}{b-f}&space;&&space;0&space;&&space;0&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;\frac{2}{b-f}&space;&&space;0&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;0&space;&&space;\frac{-2}{b-f}&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;1&space;\end{bmatrix}&space;\begin{bmatrix}&space;x&space;\\&space;y\\&space;z\\&space;1&space;\end{bmatrix}&space;=&space;\begin{bmatrix}&space;\frac{2}{b-f}x&space;-&space;\frac{b&plus;f}{b-f}\\&space;\frac{2}{b-f}f&space;-&space;\frac{b&plus;f}{b-f}\\&space;\frac{2}{b-f}z&space;-&space;\frac{b&plus;f}{b-f}\\&space;1&space;\end{bmatrix}&space;\\&space;\vec{v_2}&space;=&space;\begin{bmatrix}&space;2&space;&&space;0&space;&&space;0&space;&&space;-b-f&space;\\&space;0&space;&&space;2&space;&&space;0&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;-2&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;b-f&space;\end{bmatrix}&space;\begin{bmatrix}&space;x&space;\\&space;y&space;\\&space;z&space;\\&space;1&space;\end{bmatrix}&space;=&space;\begin{bmatrix}&space;2x-b-f\\&space;2y-b-f\\&space;-2z-b-f\\&space;b-f\\&space;\end{bmatrix}&space;=&space;(b-f)\vec{v_1}&space;$$&space;\\&space;M_{final}&space;=&space;\begin{bmatrix}&space;2&space;&&space;0&space;&&space;0&space;&&space;0&space;\\&space;0&space;&&space;2&space;&&space;0&space;&&space;0&space;\\&space;0&space;&&space;0&space;&&space;-2&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;b-f&space;\end{bmatrix}" target="_blank"><img src="https://latex.codecogs.com/gif.latex?$$&space;\vec{v_1}&space;=&space;\begin{bmatrix}&space;\frac{2}{b-f}&space;&&space;0&space;&&space;0&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;\frac{2}{b-f}&space;&&space;0&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;0&space;&&space;\frac{-2}{b-f}&space;&&space;-\frac{b&plus;f}{b-f}&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;1&space;\end{bmatrix}&space;\begin{bmatrix}&space;x&space;\\&space;y\\&space;z\\&space;1&space;\end{bmatrix}&space;=&space;\begin{bmatrix}&space;\frac{2}{b-f}x&space;-&space;\frac{b&plus;f}{b-f}\\&space;\frac{2}{b-f}f&space;-&space;\frac{b&plus;f}{b-f}\\&space;\frac{2}{b-f}z&space;-&space;\frac{b&plus;f}{b-f}\\&space;1&space;\end{bmatrix}&space;\\&space;\vec{v_2}&space;=&space;\begin{bmatrix}&space;2&space;&&space;0&space;&&space;0&space;&&space;-b-f&space;\\&space;0&space;&&space;2&space;&&space;0&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;-2&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;b-f&space;\end{bmatrix}&space;\begin{bmatrix}&space;x&space;\\&space;y&space;\\&space;z&space;\\&space;1&space;\end{bmatrix}&space;=&space;\begin{bmatrix}&space;2x-b-f\\&space;2y-b-f\\&space;-2z-b-f\\&space;b-f\\&space;\end{bmatrix}&space;=&space;(b-f)\vec{v_1}&space;$$&space;\\&space;M_{final}&space;=&space;\begin{bmatrix}&space;2&space;&&space;0&space;&&space;0&space;&&space;0&space;\\&space;0&space;&&space;2&space;&&space;0&space;&&space;0&space;\\&space;0&space;&&space;0&space;&&space;-2&space;&&space;-b-f&space;\\&space;0&space;&&space;0&space;&&space;0&space;&&space;b-f&space;\end{bmatrix}" title="$$ \vec{v_1} = \begin{bmatrix} \frac{2}{b-f} & 0 & 0 & -\frac{b+f}{b-f} \\ 0 & \frac{2}{b-f} & 0 & -\frac{b+f}{b-f} \\ 0 & 0 & \frac{-2}{b-f} & -\frac{b+f}{b-f} \\ 0 & 0 & 0 & 1 \end{bmatrix} \begin{bmatrix} x \\ y\\ z\\ 1 \end{bmatrix} = \begin{bmatrix} \frac{2}{b-f}x - \frac{b+f}{b-f}\\ \frac{2}{b-f}f - \frac{b+f}{b-f}\\ \frac{2}{b-f}z - \frac{b+f}{b-f}\\ 1 \end{bmatrix} \\ \vec{v_2} = \begin{bmatrix} 2 & 0 & 0 & -b-f \\ 0 & 2 & 0 & -b-f \\ 0 & 0 & -2 & -b-f \\ 0 & 0 & 0 & b-f \end{bmatrix} \begin{bmatrix} x \\ y \\ z \\ 1 \end{bmatrix} = \begin{bmatrix} 2x-b-f\\ 2y-b-f\\ -2z-b-f\\ b-f\\ \end{bmatrix} = (b-f)\vec{v_1} $$ \\ M_{final} = \begin{bmatrix} 2 & 0 & 0 & 0 \\ 0 & 2 & 0 & 0 \\ 0 & 0 & -2 & -b-f \\ 0 & 0 & 0 & b-f \end{bmatrix}" /></a>